cg/services/run_devices/pacbio/metrics_parser/models.py

```diff
@@ -128,6 +122,39 @@ def set_sequencing_completed_at(cls, data: Any):
         return data
 
 
+class BarcodeMetrics(RunMetrics):
+    """Model that holds barcode data."""
+
+    barcoded_hifi_reads: int = Field(..., alias=BarcodeMetricsAliases.BARCODED_HIFI_READS)
+    barcoded_hifi_reads_percentage: Annotated[float, BeforeValidator(fraction_to_percent)] = Field(
+        ..., alias=BarcodeMetricsAliases.BARCODED_HIFI_READS_PERCENTAGE
+    )
+    barcoded_hifi_yield: int = Field(..., alias=BarcodeMetricsAliases.BARCODED_HIFI_YIELD)
+    barcoded_hifi_yield_percentage: Annotated[float, BeforeValidator(fraction_to_percent)] = Field(
+        ..., alias=BarcodeMetricsAliases.BARCODED_HIFI_YIELD_PERCENTAGE
+    )
+    barcoded_hifi_mean_read_length: int = Field(
+        ..., alias=BarcodeMetricsAliases.UNBARCODED_HIFI_MEAN_READ_LENGTH
+    )
+    unbarcoded_hifi_reads: int = Field(..., alias=BarcodeMetricsAliases.UNBARCODED_HIFI_READS)
+    unbarcoded_hifi_yield: int = Field(..., alias=BarcodeMetricsAliases.UNBARCODED_HIFI_YIELD)
+    unbarcoded_hifi_mean_read_length: int = Field(
+        ..., alias=BarcodeMetricsAliases.UNBARCODED_HIFI_MEAN_READ_LENGTH
+    )
+
+
+class SampleMetrics(RunMetrics):
+    """Model that holds run data for a specific sample."""
+
+    barcode_name: str = Field(..., alias=SampleMetricsAliases.BARCODE_NAME)
```

Comment:
rank alphabetically or in order of importance? Whichever we choose, let's choose it for all models

Reply:
dismiss, implement later